src/Sulu/Bundle/CategoryBundle/Category/KeywordManager.php

```diff
@@ -141,12 +145,14 @@ public function delete(Keyword $keyword, Category $category)
     {
         $categoryTranslation = $category->findTranslationByLocale($keyword->getLocale());
 
-        $keyword->removeCategoryTranslation($categoryTranslation);
-        $categoryTranslation->removeKeyword($keyword);
+        if ($categoryTranslation) {
```

Comment:
Create a new category, then change the locale and try creating a keyword before saving the title in that locale.